cico_do_docker_build_tag_push.sh

```diff
@@ -12,12 +12,14 @@ upstreamCheRepoFullPath=`pwd`
 UPSTREAM_TAG=$(git rev-parse --short HEAD)
 
 # Now lets build the local docker images
-mkdir ${currentDir}/target/docker
+mkdir ${currentDir}/target/docker 2>/dev/null
 cp -R dockerfiles ${currentDir}/target/docker
 
 cd ${currentDir}/target/docker/dockerfiles/che
 cat Dockerfile.centos > Dockerfile
 
+createTags=""
```

Comment:
it's used in line [88](https://github.com/redhat-developer/rh-che/pull/112/files#diff-6aee2f002a3f356e002c5ef80242643cR88), to show, at the end of the command, a summary of what docker images were tagged and in which environment.